packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/cloudwatch_logger_plugin.dart

```diff
@@ -273,7 +276,15 @@ class CloudWatchLoggerPlugin extends AWSLoggerPlugin
       return false;
     }
     final loggingConstraint = _getLoggingConstraint();
-    return logEntry.level >= loggingConstraint.defaultLogLevel;
+    LogLevel defaultLogLevel;
+    try {
+      defaultLogLevel = LogLevel.values
+          .byName(loggingConstraint.defaultLogLevel.toLowerCase());
```

Comment:
Nah I would throw here - we shouldn't accept invalid configurations. Users may have had a typo or something silly.

Reply:
Throw a `ConfigurationError` type

Reply:
updated the logging config to use enum instead of string. so if the config file is not valid it throws at the configuration.